herd/tests/instructions/C/LB000.litmus

```diff
@@ -0,0 +1,25 @@
+C LB000
```

Comment:
If our intention is to test `const`, I think `LB` is a very complex litmus test. Would it make sense to add a litmus test with a single write to a `const` location? We can also add other tests that aim that assert expectations from `stripe_const` and other `stripe_*` functions that change in this PR (and tbh, weren't clear even before this PR).

Reply:
I am just adding parsing support in this patch, I would be happy to add source semantics support for const too, but I would need to add a new cat file with the const semantics like I implemented for aarch64. Patch starts to runaway here

Reply:
Good point and no need to add a cat file. In which case, I think the test that demonstrates what currently works is a litmus with a single load, maybe?

`LB`, apart from being complex and testing many more things (than the parsing of `const`), also demonstrates the lack of semantics for `const` as LB with `const` annotation shows the same executions as LB without `const` annotations.

Reply:
yeah I agree, I'll put together a simple load test